### PR TITLE
Bug/move help labels

### PR DESCRIPTION
**Pull request by msarauer**

### CODE REVIEW

**Comment by mikepspc** [comment]
@msarauer This looks good.  Perhaps remove the 'pl-1' from the help text for the Origin and Destination fields.

I'd also touch base with Beth and see if the format labels make sense.  The screen reader will read "Start date. Date format: YYYY-MM-DD".  Might need to rethink that phrasing now that it will all be read as one label.

Another consideration, should we have (required) on these fields?  Probably yes.  Can be done in Prismic though.

File: src/components/input-datalist.js [hunk 1]
```diff
@@ -49,7 +49,9 @@ const InputDatalist = ({validationWarnings, setValidationWarnings, label, name,
 
     return (
         <div className="mb-4">
-            <label htmlFor={`autocomplete-${name}`}>{label}</label>
+            <label htmlFor={`autocomplete-${name}`}>{label}
+                <small id={`autocomplete-help-${name}`} className="form-text pl-1">{localeCopy.autocomplete_help_text}</small>
```
Comment (mikepspc) [inline src/components/input-datalist.js]: Removing the 'pl-1' allows the fields to look more consistent with the below date fields.

**Comment by mikepspc** [comment]
Also worth considering applying this to the email modal form.

**Comment by msarauer** [comment]
Applied changes to the email modal.

**Pull request merged**